packages/jest-snapshot/src/index.ts

```diff
@@ -463,12 +543,57 @@ export const toThrowErrorMatchingInlineSnapshot: MatcherFunctionWithContext<
   );
 };
 
+export const toThrowErrorMatchingNamedSnapshot: MatcherFunctionWithContext<
+  Context,
+  [snapshotName: string, fromPromise?: boolean]
+> = function (received: unknown, snapshotName: unknown, fromPromise: unknown) {
```

Comment:
I think it is fine to keep `fromPromise` like this since it isn’t passed by the user. Otherwise build will fail.

```suggestion
> = function (received: unknown, snapshotName: unknown, fromPromise) {
```